inspire/modules/migrator/tasks.py

```diff
@@ -124,10 +125,12 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
                 before_record_index.send(recid, json=json)
                 json.update({'_index': index, '_type': 'record', '_id': recid})
                 records_to_index.append(json)
-            except Exception:
+            except Exception as err:
+                current_app.logger.exception(err)
```

Comment:
Ah, I see now what you meant :dancer:  SO I guess we can just the logger then to not mix things.